rust_src/src/process.rs

```diff
@@ -556,6 +556,77 @@ pub fn internal_default_interrupt_process(
 #[rustfmt::skip]
 def_lisp_sym!(Qinternal_default_interrupt_process, "internal-default-interrupt-process");
 
+/// Kill process PROCESS. May be process or name of one.
+/// See function `interrupt-process' for more details on usage.
+#[lisp_fn(min = "0")]
+pub fn kill_process(process: LispObject, current_group: LispObject) -> LispObject {
+    unsafe {
+        process_send_signal(process, libc::SIGKILL, current_group, false);
+    }
+    process
+}
+
+/// Send QUIT signal to process PROCESS. May be process or name of one.
+/// See function `interrupt-process' for more details on usage.
+#[lisp_fn(min = "0")]
+pub fn quit_process(process: LispObject, current_group: LispObject) -> LispObject {
+    unsafe {
+        process_send_signal(process, libc::SIGQUIT, current_group, false);
+    }
+    process
+}
+
+/// Stop process PROCESS. May be process or name of one.
+/// See function `interrupt-process' for more details on usage.
+/// If PROCESS is a network or serial or pipe connection, inhibit handling
+/// of incoming traffic.
+#[lisp_fn(min = "0")]
+pub fn stop_process(process: LispObject, current_group: LispObject) -> LispObject {
+    let mut p_ref: LispProcessRef = get_process(process).into();
+    let process_type = p_ref.ptype();
+    if process_type.eq(Qnetwork) || process_type.eq(Qserial) || process_type.eq(Qpipe) {
```

Comment:
This makes me miss Python's `in` operator. I want to re-write this as `if process_type in [Qnetwork, Qserial, Qpipe]`.

Reply:
Yes, Python's `in` would look great here. :)
But I don't see the point in separate function because there are some others conditions with these values in other unported functions in process.c. We can't consider all of them in single function (unless define macros, but I don't think it's worth it).

Reply:
You could do `[Qnetwork, Qserial, Qpipe].contains(process_type)`